web/src/main/angular/src/app/core/components/server-map/class/server-map-diagram-with-cytoscapejs.class.ts

```diff
@@ -153,12 +153,40 @@ export class ServerMapDiagramWithCytoscapejs extends ServerMapDiagram {
         });
     }
 
+    getResponseInfo(linkData: ILinkInfo | IShortLinkInfo): any {
+        if (typeof linkData === "undefined" || !linkData) {
+            return "";
+        }
+        const histogram_error = linkData.histogram ? linkData.histogram.Error : 0;
+        const histogram_avg = linkData.responseStatistics ? linkData.responseStatistics.Avg :0;
+        const histogram_max = linkData.responseStatistics ? linkData.responseStatistics.Max : 0;
+        const tot = histogram_error + histogram_avg + histogram_max;
+        if (isNaN(tot) || tot <= 0) {
+            return "";
+        }
+        let hint = " (";
+        if (histogram_error > 0) {
+            hint += "err:" + histogram_error + " ";
+        }
+        if (histogram_avg > 0) {
+            hint += "avg:" + histogram_avg + "ms ";
+        }
+        if (histogram_max > 0) {
+            hint += "max:" + histogram_max + "ms ";
+        }
+        hint = hint.substring(0, hint.length - 1);
+        return hint + ") ";
+    }
+
+
```

Comment:
- Please use single quotes for string for the consistency.
- Give a space before `0` in line 161.